lib/main.js

```diff
@@ -235,10 +261,11 @@ export default {
       }
     }
 
+    const program = await this.getProgram(Linter, configurationPath);
```

Comment:
Fixed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/5de77d83408e5d656c5e994729764daafbe38304